doc/release-notes/0.43/0.43.md

```diff
@@ -56,6 +56,13 @@ The following table covers notable changes in v0.43.0. Further information about
 On OpenJDK 19 and later, the Linux reference compiler was already updated to gcc 11.2 in [release 0.37.0](version0.37.md).</td>
 </tr>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18135">#18135</a></td>
+<td valign="top">OpenJ9 Windows&reg; OpenJDK 8, 11, and 17 builds now compile with Microsoft&reg; Visual Studio 2022.</td>
+<td valign="top">OpenJDK 8, 11, and 17 (Windows)</td>
```

Comment:
It's inconsistent with the Linux change. The Linux change uses "OpenJDK 8 and later" and mentions the newer gcc version was already used for 19 and later. This is also true for Windows.

Reply:
So, should I change for Linux "OpenJDK 8, 11, and 17? And for Windows, add that this compilation was already happening for 19 and later?

Reply:
I agree.

Reply:
I have updated the release notes. Thanks!